### PR TITLE
Add install-apps.sh to install apps (user_oidc)

### PR DESCRIPTION
**Pull request by printminion-co**

### CODE REVIEW

File: installed-apps.inc.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# List of apps to be installed.
+# Managed as include file so that it can be used in various scripts.
+export ENABLED_APPS="user_oidc"
```
Comment (printminion-co) [inline installed-apps.inc.sh:3]: ooc app:install user_oidc breaks for me with "Error: Could not download app user_oidc"
probably we should download/configure as submodule beforehand

File: install-apps.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+main() {
+	if ! which jq >/dev/null 2>&1; then
+		fail "Error: jq is required"
+	fi
+
+	echo "Install and enable apps ..."
+
+	for app in ${ENABLED_APPS}; do
+		echo "Install app '${app}' ..."
+		ooc app:install "${app}"
```
Comment (printminion-co) [inline install-apps.sh:29]: We should fail if app install is not possible.
A-lá
`		if [ "${?}" -ne 0 ]; then
			fail "Error: failed to install app '${app}'"
		fi
`

**Review by printminion-co — COMMENTED** [review]
review ok with amends

**Comment by thlehmann-ionos** [comment]
Issues addressed. Added in https://github.com/IONOS-Productivity/easystorage-config/pull/2

**Pull request closed without merging**